lib/class-wp-rest-url-details-controller.php

```diff
@@ -137,6 +143,39 @@ public function parse_url_details( $request ) {
 		return apply_filters( 'rest_prepare_url_details', $response, $url, $request, $remote_url_response );
```

Comment:
Note to self. Pass the  `$xpath` value to the filter so folks can use it to query additional parts of response.